### PR TITLE
Added sample for Static Content

### PR DESCRIPTION
**Pull request by hhariri**

### CODE REVIEW

File: ktor-features/pom.xml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```
Comment (orangy) [inline ktor-features/pom.xml]: Do not change pom.xml without a reason

File: ktor-samples/ktor-samples-static-content/pom.xml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    <artifactId>ktor-samples-static-content</artifactId>
+
+    <build>
+        <sourceDirectory>src</sourceDirectory>
+        <resources>
+            <resource>
+                <directory>resources</directory>
+            </resource>
+        </resources>
+    </build>
```
Comment (orangy) [inline ktor-samples/ktor-samples-static-content/pom.xml]: This shouldn't be needed, it's specified in root pom

File: ktor-samples/ktor-samples-static-content/src/org/jetbrains/ktor/samples/staticcontent/StaticContent.kt [hunk 3]
```diff
@@ -0,0 +1,27 @@
+package org.jetbrains.ktor.samples.staticcontent
+
+import org.jetbrains.ktor.content.*
+import org.jetbrains.ktor.host.embeddedServer
```
Comment (orangy) [inline ktor-samples/ktor-samples-static-content/src/org/jetbrains/ktor/samples/staticcontent/StaticContent.kt]: Please use `*` imports

File: ktor-features/pom.xml (hunk 1 above)
Comment (hhariri) [inline ktor-features/pom.xml]: Hmm. Wrong pom, sorry.

File: ktor-samples/ktor-samples-static-content/pom.xml (hunk 2 above)
Comment (hhariri) [inline ktor-samples/ktor-samples-static-content/pom.xml]: It's essentially copy/paste of other sample, so I guess those need to be cleaned up too.

**Pull request closed without merging**